dbt_copilot_helper/COMMANDS.md

```diff
@@ -604,7 +604,7 @@ copilot-helper domain check-domain --domain-profile (dev|live) --project-profile
 - `--help <boolean>` _Defaults to False._
   - Show this message and exit.
 
-# copilot-helper domain assign-domain
+# copilot-helper domain assign
```

Comment:
Docstring could do with an update `Check Route53 domain is pointing to the correct ECS Load Balancer.` doesn't seem quite right anymore.

Reply:
Done